<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectorCoefDHT</class>
 <widget class="QWidget" name="selectorCoefDHT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coeficientes de la Transformada Hermite</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rbOriginal">
       <property name="text">
        <string>Volumen Original</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QRadioButton" name="rb000">
       <property name="text">
        <string>000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="3">
      <widget class="QRadioButton" name="rb010">
       <property name="text">
        <string>010</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rb100">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QRadioButton" name="rb001">
       <property name="text">
        <string>001</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="3">
      <widget class="QRadioButton" name="rb110">
       <property name="text">
        <string>110</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rb101">
       <property name="text">
        <string>101</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="rb020">
       <property name="text">
        <string>020</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QRadioButton" name="rb011">
       <property name="text">
        <string>011</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QRadioButton" name="rb002">
       <property name="text">
        <string>002</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3" colspan="3">
      <widget class="QRadioButton" name="rb200">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="3">
      <widget class="QRadioButton" name="rbGradient">
       <property name="text">
        <string>Gradiente</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
